scenarios/classification/25_deployment_on_azure_apps_service_and_setting_CORS_policies.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<p>The images don't seem to be included correctly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/533/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>I've checked again on the running jupyter server on my machine and they all work.</p><p> </p><p>As mentioned above, the reason why they might not be getting displayed correctly on reviewNB could be due to the relative paths used for the links and images.</p>